<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EDM</class>
 <widget class="QMainWindow" name="EDM">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: rgb(22, 22, 22)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>268</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="minimumSize">
         <size>
          <width>225</width>
          <height>65</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
	background: rgb(22, 22, 22);
	border:none;
	color:white;
	
	font: 700 35pt &quot;Segoe UI&quot;;
}</string>
        </property>
        <property name="text">
         <string>EDM</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>278</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>86</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="Button_7">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton 
{
    background: rgb(0, 255, 255);
    color: black;
    font: 26pt &quot;Segoe UI&quot;;
}

QPushButton:pressed 
{
    background: rgb(0, 177, 177);
    color: black;
    font: 26pt &quot;Segoe UI&quot;;
}
</string>
        </property>
        <property name="text">
         <string>(Q) Ride</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="Button_8">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
	background: rgb(35, 255, 0);
	color: black;
	font: 26pt &quot;Segoe UI&quot;;
}

QPushButton:pressed
{
	background: rgb(28, 185, 0);
	color: black;
	font: 26pt &quot;Segoe UI&quot;;
}</string>
        </property>
        <property name="text">
         <string>(W) Snare</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="Button_9">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
	background: rgb(255, 255, 0);
	color: black;
	font: 26pt &quot;Segoe UI&quot;;
}

QPushButton:pressed
{
	background: rgb(159, 159, 0);
	color: black;
	font: 26pt &quot;Segoe UI&quot;;
}</string>
        </property>
        <property name="text">
         <string>(E) Tom</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="Button_4">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
	background: rgb(255,0, 255);
	color: black;
	font: 26pt &quot;Segoe UI&quot;;
}

QPushButton:pressed
{
	background: rgb(177, 0, 177);
	color: black;
	font: 26pt &quot;Segoe UI&quot;;
}</string>
        </property>
        <property name="text">
         <string>(A) HitHat</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="Button_5">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
	background: rgb(0,128,255);
	color: black;
	font: 26pt &quot;Segoe UI&quot;;
}

QPushButton:pressed
{
	background: rgb(0,89,177);
	color: black;
	font: 26pt &quot;Segoe UI&quot;;
}</string>
        </property>
        <property name="text">
         <string>(S) Kick</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="Button_6">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
	background: rgb(255, 128, 0);
	color: black;
	font: 26pt &quot;Segoe UI&quot;;
}

QPushButton:pressed
{
	background: rgb(185, 92, 0);
	color: black;
	font: 26pt &quot;Segoe UI&quot;;
}</string>
        </property>
        <property name="text">
         <string>(D) Open</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="Button_2">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
	background: rgb(0,255,128);
	color: black;
	font: 26pt &quot;Segoe UI&quot;;
}

QPushButton:pressed
{
	background: rgb(0, 177, 89);
	color: black;
	font: 26pt &quot;Segoe UI&quot;;
}</string>
        </property>
        <property name="text">
         <string>(X) Crash</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="Button_3">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton
{
	background: rgb(255, 0, 0);
	color: black;
	font: 26pt &quot;Segoe UI&quot;;
}

QPushButton:pressed
{
	background: rgb(177, 0, 0);
	color: black;
	font: 26pt &quot;Segoe UI&quot;;
}</string>
        </property>
        <property name="text">
         <string>(C) Fill</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
